tests/webtests/EntityCreateOidcTest.php

```diff
@@ -172,7 +172,7 @@ public function test_it_forwards_to_edit_action_when_publish_failed()
         // Publish json
         $this->testMockHandler->append(new Response(200, [], '{"id":"f1e394b2-08b1-4882-8b32-43876c15c743"}'));
         // Push to Manage
-        $this->testMockHandler->append(new Response(404, [], '{"status":"failed"}'));
+        $this->testMockHandler->append(new Response(200, [], '{"status":"failed"}'));
```

Comment:
Does manage actually return a 200 response code upon failure?

Reply:
Earlier on this was a 404 but that seemed to be never returned. Changed it to a 400  to reflect the situation better because that's what I'm currently getting on failure from manage.